test/export/StructureDefinitionExporter.test.ts

```diff
@@ -1198,6 +1198,55 @@ describe('StructureDefinitionExporter', () => {
     expect(loggerSpy.getLastMessage()).toMatch(/File: Fixed\.fsh.*Line: 4\D*/s);
   });
 
+  it('should log an error when a fixed value on a child interferes with a fixed slice on the parent', () => {
+    // Profile: ParentWithFixedSlice
+    // Parent: Observation
+    // * category ^slicing.discriminator[0].type = #pattern
+    // * category ^slicing.discriminator[0].path = "code"
```

Comment:
It's just an example, but technically this path is not valid in this context.  The discriminator path should be relative to the element that is being sliced.  In this case, since `category` is a `CodeableConcept`, `"code"` doesn't work (because `CodeableConcept.code` isn't a thing).  Since the pattern is actually applied to `category` itself, you really want the path to be `"$this"`.